test/first_order_sequence.jl

```diff
@@ -137,7 +137,7 @@ end
     C = Matrix(DataFrame(CSV.File(joinpath(path, "$(file_prefix)_C.csv"); header = false)))
     D_raw = Matrix(DataFrame(CSV.File(joinpath(path, "$(file_prefix)_D.csv");
                                       header = false)))
-    D = TuringDiagMvNormal(zero(vec(D_raw)), vec(D_raw))
+    D = MvNormal(zero(vec(D_raw)), vec(D_raw))
```

Comment:
Construct with the Mean zero form descirvbed in #101 .  e.g. `MvNormal(Diagonal(D_raw))` or something like that instead.

Reply:
Oh disregard, this is just the test.  Sorry!